src/pages/Device.js

```diff
@@ -0,0 +1,38 @@
+import * as React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+
+export default function Device(props) {
+  const { class_name, launch } = props
+  return (
+
+    <span className={class_name}>
+      <StaticImage
+        src="../images/device.png"
+        width={360}
+        quality={100}
+        alt="device"
+        loading="eager"
+        className="device-frame"
+      />
+      {launch == 'true' ?
+        <StaticImage
```

Comment:
The only thing needing to change on these is the `src` right? No need to duplicate the entire component. DRY.